docs: directions for meshery icons and static files in UI (#6726)
meshery/meshery: docs/pages/project/contributing/contributing-ui.md

All of the above steps would get the Meshery's development server running for you to work on in any IDE of your choice.

> ### Meshery static files and icons

Reviewer: ```suggestion
### Static files and icons
```

The Meshery public folder contains static files and its folder structure may look like this:
```
meshery
└── ui
    └── public
        └── static
            ├── favicon.png
            ├── fonts
            ├── img
            └── style
```

> 

Reviewer: @Krishna2323 look good?

##### Conventions for svg files
1. svg icons should be optimized and compressed, there are online [svg optimizers and viewers](https://www.svgviewer.dev/) that can compress the svg files to smaller size.
2. All svg icons should have "height" and "width" properties set to some value. If it can be possible try to give the icons the default height and width according to their needs in UI, e.g. if its a small icons `20px*20px` is a good choice, if its medium `24px*24px` works well, and accordingly.
3. Always include this XML header in each SVG image:
      ```
      <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg>
      ```
4. Svg can only fall under two categories, and this categories should be the name of folder \
    1. white: containg white or mono-colored version of that svg
    2. colored: containing colored version of that svg. \
    
    e.g.: for meshery icon, the colored and white version's folder structure may look like this
    ```
    └── img
        └── meshery
            ├── white
            |   └── meshery.svg

>             └── colored

Reviewer: ```suggestion
            └── color
```

> #### Meshery Icons and Images
> 
> All the images or icons needs to be sourced from the [public directory of images](https://github.com/meshery/meshery/tree/master/ui/public/static/img). The files written inside this directory should only end with the extensions like `.svg`, `.png`, `.jpg` or `.jpeg`. Among all the `.svg` and `.png` are most favourable for usage.
> 
> ##### Conventions for svg files
> 1. svg icons should be optimized and compressed, there are online [svg optimizers and viewers](https://www.svgviewer.dev/) that can compress the svg files to smaller size.
> 2. All svg icons should have "height" and "width" properties set to some value. If it can be possible try to give the icons the default height and width according to their needs in UI, e.g. if its a small icons `20px*20px` is a good choice, if its medium `24px*24px` works well, and accordingly.
> 3. Always include this XML header in each SVG image:
>       ```
>       <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg>
>       ```
> 4. Svg can only fall under two categories, and this categories should be the name of folder \
>     1. white: containg white or mono-colored version of that svg
>     2. colored: containing colored version of that svg. \
>     
>     e.g.: for meshery icon, the colored and white version's folder structure may look like this
>     ```
>     └── img
>         └── meshery
>             ├── white
>             |   └── meshery.svg
>             └── colored

Reviewer: ```suggestion
#### Meshery Icons and Images

All the images or icons are stored in the [public/static/img directory](https://github.com/meshery/meshery/tree/master/ui/public/static/img). The files written inside this directory should only end with the extensions like `.svg`, `.png`, `.jpg` or `.jpeg`. Above all file formats SVG is preferred with rasterized image formats being a secondary choice when no vector-based format is available.

##### Conventions for SVG files
1. SVG icons should be optimized and compressed. Use an [SVG optimizer], like [SVG Viewer](https://www.svgviewer.dev), to compress SVG files to their most compact form.
2. All SVG icons should have "height" and "width" properties set to some value. If it can be possible try to give the icons the default height and width according to their needs in UI, e.g. if its a small icons `20px*20px` is a good choice, if its medium `24px*24px` works well, and so on.
3. Always include this XML header in each SVG image:
      ```
      <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg>
      ```
4. Each icon needs to be available as in two color modes: color and white. Icons should be persistent under their respective folder name.
    1. white: containing a white or mono-colored version of that icon/svg.
    2. color: containing a colored version of that icon/svg. 
    
  Using the Meshery icon as an example, the folder structure should be as follows.

    ```
    └── img
        └── meshery
            ├── white
            |   └── meshery.svg
            └── color
```

> #### Meshery Icons and Images
> 
> All the images or icons needs to be sourced from the [public directory of images](https://github.com/meshery/meshery/tree/master/ui/public/static/img). The files written inside this directory should only end with the extensions like `.svg`, `.png`, `.jpg` or `.jpeg`. Among all the `.svg` and `.png` are most favourable for usage.
> 
> ##### Conventions for svg files
> 1. svg icons should be optimized and compressed, there are online [svg optimizers and viewers](https://www.svgviewer.dev/) that can compress the svg files to smaller size.
> 2. All svg icons should have "height" and "width" properties set to some value. If it can be possible try to give the icons the default height and width according to their needs in UI, e.g. if its a small icons `20px*20px` is a good choice, if its medium `24px*24px` works well, and accordingly.
> 3. Always include this XML header in each SVG image:
>       ```
>       <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg>
>       ```
> 4. Svg can only fall under two categories, and this categories should be the name of folder \
>     1. white: containg white or mono-colored version of that svg
>     2. colored: containing colored version of that svg. \
>     
>     e.g.: for meshery icon, the colored and white version's folder structure may look like this
>     ```
>     └── img
>         └── meshery
>             ├── white
>             |   └── meshery.svg
>             └── color
>                 └── meshery.svg

Author: ```suggestion
#### Meshery Icons and Images
All the images or icons are stored in the [public/static/img directory](https://github.com/meshery/meshery/tree/master/ui/public/static/img). The files written inside this directory should only end with the extensions like `.svg`, `.png`, `.jpg` or `.jpeg`. Above all file formats SVG is preferred with rasterized image formats being a secondary choice when no vector-based format is available.
##### Conventions for SVG files
1. SVG icons should be optimized and compressed. Use an [SVG optimizer], like [SVG Viewer](https://www.svgviewer.dev), to compress SVG files to their most compact form.
2. All SVG icons should have "height" and "width" properties set to some value. If it can be possible try to give the icons the default height and width according to their needs in UI, e.g. if its a small icons `20px*20px` is a good choice, if its medium `24px*24px` works well, and so on.
3. Always include this XML header in each SVG image:
      ```
      <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg>
      ```
4. Each icon needs to be available as in two color modes: color and white. Icons should be persistent under their respective folder name.
    1. white: containing a white or mono-colored version of that icon/svg.
    2. color: containing a colored version of that icon/svg. 
    
  Using the Meshery icon as an example, the folder structure should be as follows.
    ```
    └── img
        └── meshery
            ├── white
            |   └── meshery.svg
            └── color
#### Meshery Icons and Images

All the images or icons needs to be sourced from the [public directory of images](https://github.com/meshery/meshery/tree/master/ui/public/static/img). The files written inside this directory should only end with the extensions like `.svg`, `.png`, `.jpg` or `.jpeg`. Among all the `.svg` and `.png` are most favourable for usage.

##### Conventions for svg files
1. svg icons should be optimized and compressed, there are online [svg optimizers and viewers](https://www.svgviewer.dev/) that can compress the svg files to smaller size.
2. All svg icons should have "height" and "width" properties set to some value. If it can be possible try to give the icons the default height and width according to their needs in UI, e.g. if its a small icons `20px*20px` is a good choice, if its medium `24px*24px` works well, and accordingly.
3. Always include this XML header in each SVG image:
      ```
      <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg>
      ```
4. Svg can only fall under two categories, and this categories should be the name of folder \
    1. white: containg white or mono-colored version of that svg
    2. colored: containing colored version of that svg. \
    
    e.g.: for meshery icon, the colored and white version's folder structure may look like this
    ```
    └── img
        └── meshery
            ├── white
            |   └── meshery.svg
            └── color
                └── meshery.svg
```

    ```

> 5. Avoid any kind of duplicacy in the versions of icon used.

Reviewer: ```suggestion
5. Avoid any kind of duplicity in the versions of icon used.
```
